common/static/common/js/components/PortfolioExperimentUpsellModal.jsx

```diff
@@ -49,6 +49,8 @@ export class PortfolioExperimentUpsellModal extends React.Component {
             </div>
         );
 
+        let upgrade_text = "Upgrade (" + this.props.currency_symbol + "100 " + this.props.currency + ")"
```

Comment:
nit:
```suggestion
        let upgrade_text = `Upgrade (${this.props.currency_symbol}100 ${this.props.currency})`
```

Reply:
Also, if this is a local variable, can we use `upgradeText` instead of `upgrade_text` ?

Reply:
done.